### ISSUE TITLE
Chrome, Windows Defender, and others flagging as virus

### ISSUE THREAD

**Issue by PicantePeperoni** [Notifiarr/notifiarr#748]
Windows defender flagged as a Trojan in its latest quick scan and quarantined the exe and process. I assume that this is due to the hidden powershell changes but might not be.

During re-download, Chome also blocked the file so I checked the SHA256 and ran it though virustotal.

https://www.virustotal.com/gui/file/1704400cac8411fd98bdef3bdb4c116f1e561671b7f11f7de3e6d79e7a87f4a2/detection

Looks like false positives but wnated to make sure you are aware.

**Comment by davidnewhall** [comment]
WIndows sucks. :) If I used it, I might be able to fix this junk.

**Comment by PicantePeperoni** [comment]
Thats what I assumed also and thank you for the details. 

For the hidden powershell, I was referring to the match with sigma rule.
"Matches rule Non Interactive PowerShell Process Spawned by Roberto Rodriguez @Cyb3rWard0g (rule), oscd.community (improvements) at Sigma Integrated Rule Set (GitHub)
Detects non-interactive PowerShell activity by looking at the "powershell" process with a non-user GUI process such as "explorer.exe" as a parent."

While I had warnings with previous versions of the software during download, it wasnt until v0.7.3 May 29, 2024 that Windows Defender took action and quarantined the process and exe. I thought I had saw a release note that this version switched to a background cmd prompt but I could be wrong about that.

**Comment by davidnewhall** [comment]
This version removed the 'background' command/terminal window. It now acts like a GUI-only process in the eyes of Windows.

**Comment by PicantePeperoni** [comment]
Makes sense. I was trying to draw a pattern between this update and windows. All of the old version EXEs are in that same folder and windows defender doesn't have any problem with them. But maybe defender updated it's definitions and maybe that's why. Only windows knows what it does, we can only guess.

**Comment by davidnewhall** [comment]
I'll close this when I sign the windows exe file and remove the nasty Microsoft warnings.

**Comment by davidnewhall** [comment]
Hello @PicantePeperoni!

Thanks for your patience. 

Can you possibly download the `exe.zip` file found [here](https://unstable.notifiarr.app)? And let me know how badly Windows freaks out when you decompress it. :)

**Comment by davidnewhall** [comment]
Awesome! This is exactly the report I was hoping for. Thanks so much for testing that out!! The popup you got goes away with time, or with more money (more expensive signing cert).

**Comment by davidnewhall** [comment]
Also, what version of Windows are you testing with?

**Comment by PicantePeperoni** [comment]
Im on the latest Windows 11 23H2
